addons/web/controllers/main.py

```diff
@@ -2067,27 +2067,30 @@ def report_routes(self, reportname, docids=None, converter=None, **data):
     # Misc. route utils
     #------------------------------------------------------
     @http.route(['/report/barcode', '/report/barcode/<type>/<path:value>'], type='http', auth="public")
-    def report_barcode(self, type, value, width=600, height=100, humanreadable=0, quiet=1, mask=None):
+    def report_barcode(self, type, value, **kwargs):
         """Contoller able to render barcode images thanks to reportlab.
         Samples:
```

Comment:
While at it, this should be preceded and followed by an empty line, and should end with `::` to trigger a code block.